src/import-data/ImportDataDestination.test.ts

```diff
@@ -77,17 +65,42 @@ describe('ImportDataDestination', () => {
     expect(isWarningShown).toEqual(isProtectedData);
   });
 
-  it('should disable noncompliant workspaces', async () => {
+  it('should filter workspaces through canImportIntoWorkspace', async () => {
```

Comment:
Tests for the filter logic are now in `import-utils.test.ts`, so this test can use a mock implementation. It just needs to verify that the component is using the filter.